src/test/regress/bin/normalize.sed

```diff
@@ -163,9 +163,12 @@ s/failed to roll back prepared transaction '.*'/failed to roll back prepared tra
 
 # Table aliases for partitioned tables in explain outputs might change
 # regardless of postgres appended an _int suffix to alias, we always append _xxx suffix
-s/(->\ +Seq Scan on\ +)(.*)(_[0-9]+)(_[0-9]+) \2(_[0-9]+|_xxx)?/\1\2\3\4 \2_xxx/g
+# "->  <scanMethod> Scan on <tableName>_<partitionId>_<shardId> <tableName>_<aliasId>" and
+# "->  <scanMethod> Scan on <tableName>_<partitionId>_<shardId> <tableName>" becomes
+# "->  <scanMethod> Scan on <tableName>_<partitionId>_<shardId> <tableName>_xxx"
+s/(->.*Scan on\ +)(.*)(_[0-9]+)(_[0-9]+) \2(_[0-9]+|_xxx)?/\1\2\3\4 \2_xxx/g
```

Comment:
why do we have __xxx in `(_[0-9]+|_xxx)` ?

Reply:
>"[0-9]+|_xxx)?"

As there is a question mark here, `normalize.sed` would put _xxx each time we execute it if do not catch existing "_xxx".

Why question mark ?
Because postgres might not append `_int` suffix to partition table alias